webapp/graphite/storage.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, finders=None, hosts=None):
 
     if hosts is None:
       hosts = settings.CLUSTER_SERVERS
-    remote_hosts = [host for host in hosts if not is_local_interface(host)]
+    remote_hosts = [host for host in hosts if not settings.REMOTE_EXCLUDE_LOCAL or not is_local_interface(host)]
```

Comment:
This feels like a weird `if not ... or not` to me. Can you talk out the intent here?